openedge-checks/src/main/java/org/sonar/plugins/openedge/api/LicenceRegistrar.java

```diff
@@ -1,7 +1,10 @@
 package org.sonar.plugins.openedge.api;
 
 import org.sonar.api.batch.BatchSide;
+import org.sonar.api.batch.ScannerSide;
+import org.sonarsource.api.sonarlint.SonarLintSide;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonarsource.api.sonarlint.SonarLintSide'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)